### PR TITLE
 Correct the description in .shed.yml for IDR

### PR DESCRIPTION
**Pull request by lparsons**
The IDR long description was not correct and made discovery in the toolshed more difficult.

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/idr/.shed.yml [hunk 1]
```diff
@@ -2,13 +2,13 @@ categories:
 - Sequence Analysis
 homepage_url: https://github.com/nboley/idr
 long_description: |
-    idr is a program for quantifying abundances of transcripts from RNA-Seq
-    data, or more generally of target sequences using high-throughput sequencing
-    reads. It is based on the novel idea of pseudoalignment for rapidly
-    determining the compatibility of reads with targets, without the need for
-    alignment.
+    The IDR (Irreproducible Discovery Rate) framework is a uniﬁed approach to measure
```
Comment (nsoranzo) [inline tools/idr/.shed.yml:5]:
```suggestion
    The IDR (Irreproducible Discovery Rate) framework is a unified approach to measure
```

File: tools/idr/.shed.yml [hunk 2]
```diff
@@ -2,13 +2,13 @@ categories:
...
+    the reproducibility of ﬁndings identiﬁed from replicate experiments and provide
```
Comment (nsoranzo) [inline tools/idr/.shed.yml:6]:
```suggestion
    the reproducibility of findings identified from replicate experiments and provide
```

File: tools/idr/.shed.yml [hunk 3]
```diff
@@ -2,13 +2,13 @@ categories:
...
+    highly stable thresholds based on reproducibility. Unlike the usual scalar measures
+    of reproducibility, the IDR approach creates a curve, which quantitatively assesses
+    when the ﬁndings are no longer consistent across replicates.
```
Comment (nsoranzo) [inline tools/idr/.shed.yml:9]:
```suggestion
    when the findings are no longer consistent across replicates.
```

**Review by nsoranzo — COMMENTED** [review]
Can you also please fix the `fi` in the `<help/>` in tools/idr/idr.xml ?

**Comment by lparsons** [comment]
Thanks for catching those @nsoranzo. Should be fixed now.

File: tools/idr/idr.xml [hunk 4]
```diff
@@ -126,7 +126,7 @@
      </tests>
     <help>
 <![CDATA[
-The `IDR <https://github.com/nboley/idr/blob/master/README.md>`__ (Irreproducible Discovery Rate) framework is a uniﬁed approach to measure the reproducibility of ﬁndings identiﬁed from replicate experiments and provide highly stable thresholds based on reproducibility. Unlike the usual scalar measures of reproducibility, the IDR approach creates a curve, which quantitatively assesses when the ﬁndings are no longer consistent across replicates.
+The `IDR <https://github.com/nboley/idr/blob/master/README.md>`__ (Irreproducible Discovery Rate) framework is a uniﬁed approach to measure the reproducibility of ﬁndings identiﬁed from replicate experiments and provide highly stable thresholds based on reproducibility. Unlike the usual scalar measures of reproducibility, the IDR approach creates a curve, which quantitatively assesses when the findings are no longer consistent across replicates.
```
Comment (nsoranzo) [inline tools/idr/idr.xml:129]:
```suggestion
The `IDR <https://github.com/nboley/idr/blob/master/README.md>`__ (Irreproducible Discovery Rate) framework is a unified approach to measure the reproducibility of findings identified from replicate experiments and provide highly stable thresholds based on reproducibility. Unlike the usual scalar measures of reproducibility, the IDR approach creates a curve, which quantitatively assesses when the findings are no longer consistent across replicates.
```

**Comment by lparsons** [comment]
Geesh! Sorry, I missed some of them I guess... Maybe need to grep or something.

**Review by nsoranzo — APPROVED** [review]

**Comment by nsoranzo** [comment]
> Geesh! Sorry, I missed some of them I guess... Maybe need to grep or something.

Good thinking! Found these, if you are up for it:

tools/deseq2/deseq2.xml
179:            help=" DESeq2 performs independent ﬁltering by default using the mean of normalized counts as a ﬁlter statistic" />
209:            help=" DESeq2 performs independent ﬁltering by default using the mean of normalized counts as a ﬁlter statistic" />

tools/exomedepth/exomedepth.xml
114:The ideas used in this package are of course not speciﬁc to exome sequencing and could be applied to other

**Pull request merged**